mlflow/utils/rest_utils.py

```diff
@@ -77,6 +77,13 @@ def http_request_safe(host_creds, endpoint, **kwargs):
     Wrapper around ``http_request`` that also verifies that the request succeeds with code 200.
     """
     response = http_request(host_creds=host_creds, endpoint=endpoint, **kwargs)
+    return http_response_handler(response, endpoint)
+
+
+def http_response_handler(response, endpoint):
+    """
+    Handler for ``http_request`` returned response verifies the request succeeds with code 200.
```

Comment:
response that verifies

Reply:
Suggestion (nit): """Verify the return code and raise exception if the request was not successful."""